doonut-app-external-api/src/main/kotlin/com/doonutmate/oauth/service/OauthService.kt

```diff
@@ -46,4 +52,12 @@ class OauthService(
         }
         return memberId
     }
+
+    private fun createDefaultCalendar(memberId: Long) {
+        val fetchedMember: Member = memberBusinessService.get(memberId)
```

Comment:
변수명 그냥 member로 써도 될 것 같아요!
```suggestion
        val member: Member = memberBusinessService.get(memberId)
```